Plugins/Public/playercntl_plugin/MiscCmds.cpp

```diff
@@ -64,6 +64,13 @@ namespace MiscCmds
 	/// Cost to drop reputation changes.
 	int set_iRepdropCost = 0;
 
+	/// Cost to drop all reputation changes.
+	int set_iResetrepCost = 0;
+	// Resetrep is not allowed until server restart
```

Comment:
this doc comment is not related to the variable below it

Reply:
Fixed!